app/views/home/_system.html.haml

```diff
@@ -8,23 +8,26 @@
           .home-icon-box
             %a.search{href: "/shops"}
           .home-icon-box-bottom
-            %h4
-              = t :system_step1
+            %a.search{href: "/shops"}
```

Comment:
Why did you add those CSS classes? They seem unnecessary.

Reply:
Just to be consistent with icons, but you are right these classes don't provide any value. I will delete and update my PR